bazel/cc_wrapper.py

```diff
@@ -28,20 +28,22 @@ def sanitize_flagfile(in_path, out_fd):
         os.write(out_fd, "-lc++\n")
 
 
+# Is the arg a flag indicating that we're building for C++ (rather than C)?
+def is_cpp_flag(arg):
+  return arg in [
+      "-static-libstdc++", "-stdlib=libc++", "-std=c++0x", "-std=c++",
+      "-std=gnu++", "-lstdc++", "-lc++"
```

Comment:
Sorry if what I said was confusing, but when I said "starting with `-std=c++` and `-std=gnu++`", I meant `startswith()`, so that it would cover `-std=c++0x`, `-std=c++1x`, `-std=c++11`, `-std=gnu++11`, etc.

You need to change this to:
```
def is_cpp_flag(arg):
  return arg in ["-static-libstdc++", "-stdlib=libc++", "-lstdc++", "-lc++"
                ] or arg.startswith("-std=c++") or arg.startswith("-std=gnu++")
```
Thanks!